CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),  
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) from version [0.1.0] moving forward.
 
+## [0.2.1] - 2025-06-28
+### Fixed
+- Bug fix targeting issue ([#53](https://github.com/fable-compiler/vite-plugin-fable/issues/53)) as log warn and no crash, for protocol error in json rpc serialization when there is id count mismatch
```

Comment:
This changelog entry lacks consistency. Please adhere to a uniform format and avoid trends.

Additionally, it appears that not many fixes are being made. This could be misleading for end-users, so I would prefer to follow the updates in your other PR.